src/main/java/com/ibm/watson/modelmesh/payload/PayloadProcessor.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2023 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.ibm.watson.modelmesh.payload;
+
+public interface PayloadProcessor {
+
+    String getName();
+
+    void processRequest(Payload payload);
+
+    void processResponse(Payload payload);
```

Comment:
Thinking it might be much simpler to just have a single `processPayload` method? Could have a boolean to indicate whether it's a request or response... that should probably just be a flag in the payload itself anyhow right?

Reply:
good point, I'll introduce a `String` field `kind` that indicates the kind of payload (e.g., request, response)